classes/user_extractor.php

```diff
@@ -55,7 +56,8 @@ public static function get_user(string $fieldname, string $fieldvalue) {
 
             $joins = " LEFT JOIN {user_info_field} f ON f.shortname = :fieldname ";
             $joins .= " LEFT JOIN {user_info_data} d ON d.fieldid = f.id AND d.userid = u.id ";
-            $fieldsql = " AND d.data = :fieldvalue";
+            $datafield = $insensitive ? "LOWER(d.data)" : "d.data";
```

Comment:
If you are comparing case insensitive wouldn't you want to lower() both sides of the comparison?

Reply:
We're already ensuring that `$fieldvalue` is lower case from the caller (I suppose it's only a few extra cycles and molecules of CO₂ to check in SQL too). Thoughts?

Reply:
Actually, yeah let's fix it in SQL...